test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -174,71 +174,65 @@ Describe "Get-Content" -Tags "CI" {
   }
 
   It "Should throw TailAndHeadCannotCoexist when both -Tail and -TotalCount switches are used" {
-    try {
-      Get-Content -path $testPath -Tail 1 -TotalCount 1 -ea stop
-      throw "expected exception was not delivered"
-    }
-    catch {
-      $_.FullyQualifiedErrorId | should be "TailAndHeadCannotCoexist,Microsoft.PowerShell.Commands.GetContentCommand"
-    }
+    { 
+      Get-Content -Path $testPath -Tail 1 -TotalCount 1 -ErrorAction Stop
+    } | ShouldBeErrorId "TailAndHeadCannotCoexist,Microsoft.PowerShell.Commands.GetContentCommand"
   }
 
   It "Should throw TailNotSupported when -Tail used with an unsupported provider" {
-    pushd env:
-    try {
-      Get-Content PATH -Tail 1 -ea stop
-      throw "expected exception was not delivered"
-    }
-    catch {
-      $_.FullyQualifiedErrorId | should be "TailNotSupported,Microsoft.PowerShell.Commands.GetContentCommand"
-    }
-  	popd
+    Push-Location env:
+    {
+      Get-Content PATH -Tail 1 -ErrorAction Stop
```

Comment:
Maybe replace PATH with something like UnsupportedProvider.

Reply:
UnsupportedProvider would not be a valid environment variable and the ErrorId would be different i.e. PathNotFound instead of TailNotSupported.
Should I create one with that name before the test and then remove it?